### DISCUSSION TITLE
obs-midi-mg with Linux

### DISCUSSION THREAD

**Discussion by PhilippKuhn** [nhielost/obs-midi-mg#139] [Q&A]
Hi, I am trying to get obs studio to run with obs-midi-mg on debian 12 but it is not working. If I install the newest 3.0.3 Version, nothing happens, if I take an older version for example the 2.8., obs is not starting anymore. Any ideas for a workaround to use obs-midi-mg on debian12? Would be great!

**Comment by lhrobert** [comment]
I'm using Ubuntu 24.04, so not exactly the same setting as you. I had an issue with version 3.0.3 it would not receive MIDI message. I solved this problem by tranferring MIDI message on Midi Through Port 0 (which you should have as well on Debian). However, this might be unrelated since for me 2.8 was working fine. Just naive suggestion: are you using the last version of OBS ?
